src/components/forms/GeneralForm.vue

```diff
@@ -0,0 +1,77 @@
+<template>
```

Comment:
I would give it a more explicit name like CreateFurnitureForm (so you keep the CRUD reference language)